### ISSUE TITLE
succade + Lemonbar 1.4 = broken

### ISSUE THREAD

**Issue by domsson** [domsson/succade#51]
Something funky is going on when using the newest version of Lemonbar, 1.4. Blocks don't update anymore and we can see this going on in the terminal:

```
Invalid font index 3
Invalid font index 3
}°U
Invalid font index 3
Invalid font index 3
Invalid font index 3
```

**Comment by domsson** [comment]
Ah yes, what a stupid mistake. I have no idea how this was actually working fine before and only broke now:

We're always using "T1" (font slot 1) for the block font, "T2" for label font and "T3" for affix font. But if only one or two (or even none) of those are specified in the config, then the according indices will be invalid.

**Comment by domsson** [comment]
Fixed with v2.1.3
